p/scripts/main.js

```diff
@@ -1172,6 +1172,14 @@ function init_print_action() {
 	});
 }
 
+function init_post_action() {
+	$('.item.share > a[href="POST"]').click(function(event){
```

Comment:
I can better like with spaces before the anonymous function and before the curly bracket like:
`.click(function (event) {` but this is just a matter of taste :-)

Reply:
I prefer that as well yeah, basically a matter of taste. ;-)

Reply:
I'll change that later. I have no preference on that matter.